src/App.css

```diff
@@ -63,6 +63,26 @@ button:focus-visible {
   outline: 3px solid var(--primary-blue);
 }
 
+/* Scrollbar */
+::-webkit-scrollbar {
+  background-color: hsl(210, 13, 81);
+  width: 12px;
+}
+
+::-webkit-scrollbar:hover {
+  background-color: hsl(210, 13, 71);
+}
```

Comment:
These hover style should be applied to `::-webkit-scrollbar-thumb` for hover to take effect.

Reply:
@Ngoakor12  Thanks for the comments,  I have add the hover effect for scroll bar thumb, and removed the unrelated commit code in the App.css file. You can check now . if anything need to change please let me know !